README.md

```diff
@@ -57,16 +79,18 @@ print(result.document)
 from mindee import Client, product
 
 # Init a new client and add your custom endpoint (document)
-mindee_client = Client(api_key="my-api-key").add_endpoint(
-    account_name="john",
+mindee_client = Client(api_key="my-api-key")
+custom_endpoint = mindee_client.create_endpoint(
+    account_name="hohn",
```

Comment:
```suggestion
    account_name="john",
```

Reply:
Hohn is actually a name: https://en.wikipedia.org/wiki/Hohn

Though to be fair that was 100% a typo